pkgs/by-name/ve/version-cli/package.nix

```
@@ -0,0 +1,34 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+buildGoModule rec {
+  pname = "version-cli";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "asciimoth";
+    repo = "version";
+    rev = "v${version}";
+    hash = "sha256-TOmShw4nRyTWv2F0/p9X5WHjLSUz/dYCkp6/+NrILGo=";
+  };
+
+  vendorHash = "sha256-TGrtVY1ut76K2J/dpt3OvuXaOkAAnxFTYVl1aTii+AI=";
+
+  postInstall = ''
+    mkdir -p $out/share/man/man1
+    install -Dm644 man/version.1 $out/share/man/man1/version.1
```

```suggestion
    installManPage man/version.1
```

Needs `installShellFiles` in `nativeBuildInputs`

---

Thnx